.gitignore

```diff
@@ -100,3 +100,4 @@ CMakeSettings.json
 .DS_Store
 dbg/**
 *.wsp
+test-input
```

Comment:
```suggestion
test-input

```

Reply:
I put it there deliberately. I am using the `test-input` directory to collect formulae (e.g., those that we gave a wrong result at some point).

Reply:
It was about missing newline at the end of file 😄